packages/visual-editor/src/utils/api/nearbyLocations.tsx

```diff
@@ -36,3 +42,33 @@ export const fetchNearbyLocations = async ({
 
   return await response.json();
 };
+
+function getContentEndpointUrl(
```

Comment:
Noting that this will change since Spruce is going to add the endpoint to the document.